{{cookiecutter.github_project_name}}/js/package.json

```
@@ -31,7 +31,7 @@
     "rimraf": "^2.6.1"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^1.0.0",
+    "@jupyter-widgets/base": "^2.0.0",
```

For those that still need to support older deploys, this might be cleaner, but merging as-is
```suggestion
    "@jupyter-widgets/base": "^1.0.0 || ^2.0.0",
```